.circleci/test-deploy.yml

```diff
@@ -182,6 +182,15 @@ jobs:
           workspace: noos-prod
           plan_message: ${CIRCLE_BUILD_URL}
 
+  integration_test_trigger_pr_review:
```

Comment:
🐼 

Maybe reuse the command name in the integration test for clarity, similarly to other tests?

`integration_test_github_post_comment`